cgmes/cgmes-conformity/src/test/java/com/powsybl/cgmes/conformity/test/CgmesConformity1NetworkCatalog.java

```diff
@@ -110,7 +110,18 @@ public Network microBE(String modelId) {
                 .setB(2.1677e-5)
                 .setUcteXnodeCode("TN_Border_ST24")
                 .add();
-        be7.newCurrentLimits().setPermanentLimit(1180).add();
+        be7.newCurrentLimits().setPermanentLimit(1180)
+                .beginTemporaryLimit()
+                    .setName("BE-Line_7 - CL-0")
+                    .setValue(1312.0)
+                    .setAcceptableDuration(20)
```

Comment:
Even it’s for testing, acceptable durations are in seconds: I guess you want to set the acceptable duration to 20 minutes here? In that case, use 20 * 60.

Same remark for others.